code/modules/surgery/organs/organ_internal.dm

```diff
@@ -98,13 +98,16 @@
 
 //Checks to see if the organ is frozen from temperature
 /obj/item/organ/proc/is_cold()
-	var/freezing_objects = list(/obj/structure/closet/crate/freezer, /obj/structure/closet/secure_closet/freezer, /obj/structure/bodycontainer, /obj/item/autosurgeon)
+	var/freezing_objects = list(/obj/structure/closet/crate/freezer, /obj/structure/closet/secure_closet/freezer, /obj/structure/bodycontainer, /obj/item/autosurgeon, /obj/machinery/smartfridge/organ)
```

Comment:
make this a static list:
```
	var/static/list/freezing_objects
	if(!freezing_objects)
		freezing_objects = typecacheof(list(/obj/structure/closet/crate/freezer, /obj/structure/closet/secure_closet/freezer, /obj/structure/bodycontainer, /obj/item/autosurgeon, /obj/machinery/smartfridge/organ))
```
Or alternatively, a GLOB list if you are ok with admemes modifying it if they want.

Reply:
Is it worth making it a GLOB for 1 use?

Reply:
Actually I guess having a GLOB means it can be more useful